source/extensions/stat_sinks/hystrix/hystrix.h

```diff
@@ -34,6 +36,8 @@ struct ClusterStatsCache {
   RollingWindow total_;
   RollingWindow timeouts_;
   RollingWindow rejected_;
+
+  std::unordered_map<std::string, double> timing_;
```

Comment:
Why do we need this to be cached if we're just going to regenerate the map each time?

Also, why not make this a double to double instead by storing the quantile as a double rather than a string?

Reply:
Why do we need this to be cached if we're just going to regenerate the map each time? 
I still have an open question if it worth to do any kind of calculation over the percentile values, instead of giving the values since the beginning of the cluster's operation. Otherwise, I can pass it directly without using the cache.

why not make this a double to double instead by storing the quantile as a double rather than a string?
Just to overcome the inconsistent representation - "99.5" vs. all other values which are integers (25, 50, 99, etc..), and if I pass them as doubles, dashboard do not accept. It was convenient to handle this here.

Reply:
This is just mapping "99.5" to 99.5? and "25" to 25.0?

Unless there's a big efficiency issue I'd vote for doing this type coercion closer to where it's consumed.  Otherwise commenting why you are doing this would be really helpful.

Reply:
I don't mean in the PR; comments; I mean adding comments in the code. Document both what it's doing and also why you think this is helpful from a code-structure or performance perspective.